epochStart/bootstrap/disabled/disabledPoolsHolder.go

```diff
@@ -0,0 +1,106 @@
+package disabled
+
+import (
+	"github.com/ElrondNetwork/elrond-go/config"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever/dataPool"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever/dataPool/headersCache"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever/shardedData"
+	"github.com/ElrondNetwork/elrond-go/dataRetriever/txpool"
+	"github.com/ElrondNetwork/elrond-go/storage"
+	"github.com/ElrondNetwork/elrond-go/storage/storageUnit"
+)
+
+// PoolsHolder -
+type PoolsHolder struct {
+	transactions         dataRetriever.ShardedDataCacherNotifier
+	unsignedTransactions dataRetriever.ShardedDataCacherNotifier
+	rewardTransactions   dataRetriever.ShardedDataCacherNotifier
+	headers              dataRetriever.HeadersPool
+	miniBlocks           storage.Cacher
+	peerChangesBlocks    storage.Cacher
+	trieNodes            storage.Cacher
+	currBlockTxs         dataRetriever.TransactionCacher
+}
+
+// NewDisabledPoolsHolder -
+func NewDisabledPoolsHolder() *PoolsHolder {
```

Comment:
you can use the normal poolsholder, no need for a disabled. After start in epoch you could clean everything.

Reply:
the `NewDataPool()` constructor for a PoolsHolder requires a lot of other caches and also notifiers. It is more readable if we leave this disabled one